nova/bigvm/manager.py

```diff
@@ -142,16 +144,23 @@ def _flatten(list_of_lists):
             for p, d in provider_summaries.items():
                 used = vmware_providers.get(p, {})\
                         .get('memory_mb_used_percent', 100)
-                if used > CONF.bigvm_cluster_max_usage_percent:
+                reserved = vmware_providers.get(p, {})\
+                            .get('memory_reservable_mb_used_percent', 100)
+                if (used > CONF.bigvm_cluster_max_usage_percent
+                    or reserved > CONF.bigvm_cluster_max_reservation_percent):
```

Comment:
Please indent second lines of the `if` one more level to make it easier to see, that this line is not part of the `if`'s body.